From 13e9d66c24d1dce5179805ae5e1bf940409b4914 Mon Sep 17 00:00:00 2001
From: Mingli Yu <mingli.yu@windriver.com>
Date: Wed, 10 Aug 2022 15:21:07 +0800
Subject: [PATCH] config/nspr-config.in: don't pass LDFLAGS

Don't pass LDFLAGS to avoid exposing the build env info.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Mingli Yu <mingli.yu@windriver.com>
---
 config/nspr-config.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/nspr-config.in b/config/nspr-config.in
index 2cb62a0..2bec715 100755
--- a/config/nspr-config.in
+++ b/config/nspr-config.in
@@ -136,7 +136,7 @@ if test "$echo_libs" = "yes"; then
       if test -n "$lib_nspr"; then
 	libdirs="$libdirs -lnspr${major_version}"
       fi
-      os_ldflags="@LDFLAGS@"
+      os_ldflags="LDFLAGS"
       for i in $os_ldflags ; do
 	if echo $i | grep \^-L >/dev/null; then
 	  libdirs="$libdirs $i"
-- 
2.25.1

